src/Observable.ts

```diff
@@ -96,9 +97,10 @@ export class Observable<T> implements CoreOperators<T>  {
       }
     } else {
       const next = <((x?) => void)> observerOrNext;
-      subscriber = Subscriber.create(next, error, complete);
+      subscriber = Subscriber.create(next, error, complete, start);
     }
 
+    subscriber.start(subscriber);
     subscriber.add(this._subscribe(subscriber));
```

Comment:
Oh? For some reason I thought that was part of this feature. "Look it's unsubscribed, I should do something different (like error or notify someone)"

I'm fine with this change though, it's minimal.